src/components/CabinetModal.js

```diff
@@ -0,0 +1,87 @@
+import React from 'react'
+import { Accordion, Icon, Button, Modal, Image, Header, Grid } from 'semantic-ui-react'
+import president from "../assets/images/eboard/president.jpeg";
+import graduate from "../assets/images/eboard/graduate.jpeg";
+import secretary from "../assets/images/eboard/secretary.jpg";
+import treasurer from "../assets/images/eboard/treasurer.jpeg";
+import marketing from "../assets/images/eboard/marketing.jpeg";
+import corporate from "../assets/images/eboard/corporate.jpeg";
+import technology from "../assets/images/eboard/technology.jpeg";
+import external from "../assets/images/eboard/external.jpeg";
+import internal from "../assets/images/eboard/internal.jpeg";
+
+
+function CabinetModal(cabinet, description, cabinetEmail, pic, panels) {
+
+console.log(panels)
+
+  const [open, setOpen] = React.useState(false)
+
+
+//use Grid
```

Comment:
Delete all comments unless you think is important for other developers to know why it's there